### PR TITLE
lib/libc/math/lib_cbrtf.c : apply a coding style

### PR DESCRIPTION
**Pull request by NohYunah**
[R12] Each variable shall be declared in the new line. 
line75 : , -> ;
line76 : add static const unsigned

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [5671a6e72b9c9cad19a58a929f05ac727bb5dd64] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [5671a6e72b9c9cad19a58a929f05ac727bb5dd64] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [503c28eb0833d27e980c660f33719fc340cba0fa] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [503c28eb0833d27e980c660f33719fc340cba0fa] - Code Rule Check (C++) OK.

File: lib/libxx/libxx_new.cxx [hunk 1]
```diff
@@ -105,8 +104,8 @@ void *operator new(unsigned int nbytes)
   void *alloc = lib_malloc(nbytes);
 
 #ifdef CONFIG_DEBUG
-  if (alloc == 0)
-  {
+  if (alloc == 0) {
+
```
Comment (davidfather) [inline lib/libxx/libxx_new.cxx]: need to remove a space.

**Comment by davidfather** [comment]
Would you please re-commit this PR on 'typo_codingrule' branch instead?
In addition, before making your PRs, you need to prepare your own forked repo.
Please, refer to our contribution rules.

**Pull request closed without merging**